src/shims/manual/mscorlib.forwards.cs

```diff
@@ -20,3 +20,5 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.OrdinalComparer))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.UnitySerializationHolder))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Diagnostics.Contracts.ContractException))]
+// Genfacades tool is not able to generate a proper typeforward for System.void type.
```

Comment:
Might be more helpful comment to indicate that this requires special C# syntax that we choose to handle here.